doit/dependency.py

```diff
@@ -551,18 +608,18 @@ def get_status(self, task, tasks_dict):
 
 class JsonDependency(DependencyBase):
     """Task dependency manager with JSON backend"""
-    def __init__(self, name):
-        DependencyBase.__init__(self, JsonDB(name))
+    def __init__(self, name, **kwargs):
```

Comment:
I would prefer to avoid `kwargs` here. just put the name of the parameter!
Using the name makes it more readable.